src/avatar/avatar.component.ts

```diff
@@ -23,49 +21,42 @@ export enum Size {
     selector: "igx-avatar",
     templateUrl: "avatar.component.html"
 })
-export class IgxAvatarComponent implements AfterViewInit, AfterContentChecked {
+export class IgxAvatarComponent implements AfterContentChecked {
```

Comment:
Does it really need to implement `AfterContentChecked`? The `getRole` method seems like something that should be called once and not every detection cycle